pom.xml

```diff
@@ -2341,9 +2341,8 @@ flexible messaging model and an intuitive client API.</description>
       </snapshots>
     </repository>
     <repository>
-      <id>maven.restlet.org</id>
-      <name>maven.restlet.org</name>
-      <url>https://maven.restlet.talend.com</url>
+      <id>restlet</id>
+      <url>http://maven.icm.edu.pl/artifactory/repo/</url>
```

Comment:
The Hight version of maven has blocked the HTTP URL, you should use HTTPS instead.

Reply:
I'm using the latest Version of Maven 3.8.4, that's no problem.

Reply:
You can see here, maven has blocked the HTTP by default: https://github.com/apache/maven/commit/907d53ad3264718f66ff15e1363d76b07dd0c05f

Reply:
Well you're right, I'm using the old version of settings.xml, but when I use the new settings.xml HTTP still works, Maven use http://maven.icm.edu.pl/artifactory/repo as are the external repository? Mainly without https://maven.icm.edu.pl/artifactory/repo access.